frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -1904,6 +1900,23 @@ void ProgramStructure::createControls() {
         return;
     }
 
+    for (auto ap : action_profiles) {
+        auto action_profile = convertActionProfile(ap.first, ap.second);
+        declarations->push_back(action_profile);
```

Comment:
Should you be checking for a null `action_profile` here? Looks like `convertActionProfile()` can return null.